OpenRA.Mods.Common/Traits/Conditions/ToggleChargedConditionOnOrder.cs

```diff
@@ -18,7 +18,7 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Allow deploying on specified charge to grant a condition for a specified duration.")]
-	public class GrantConditionOnDeployWithChargeInfo : PausableConditionalTraitInfo, IRulesetLoaded
+	public class ToggleChargedConditionOnOrderInfo : PausableConditionalTraitInfo, IRulesetLoaded
```

Comment:
charging is a concept in this traits name, I think its fine

Reply:
I was looking at the updated PR's yaml portion and seeing "Toggle...OnOrder" with "ActiveCondition" below I was fully expecting to also see an order name (but ofc there isn't one). So that suffix can indeed be misleading and should probably go away.

**_HOWEVER_**, while glancing through the trait changes and specifically at `VoiceForOrder`, I got thinking that this will NOT work for more than one such trait per actor because they use the same order string. What if we solve both issues with a single fix - add an `Order` field to the TraitInfo and use that for issuing/resolving, _(For safety's sake maybe go with `$"ActivateCondition-{info.Order}"`?)_

Reply:
having multiple enabled breaks most order traits, the key is that you can have multiple by disabling one and exanbling another

Reply:
renamed to `GrantChargedConditionOnToggle`